core/embed/rust/src/ui/component/text/paragraphs.rs

```diff
@@ -279,12 +451,37 @@ struct PageBreakIterator<'a, T> {
     current: Option<PageOffset>,
 }
 
+impl<'a, T: ParagraphSource> PageBreakIterator<'a, T> {
```

Comment:
this should just be `impl<T> PageBreakIterator<'_, T>` no? as you're not using any of the type arguments in this block, it's basically just a namespace for `dyn_next`

Reply:
Oh, I didn't realize this was possible, fixed in 8a4c769cc2601aa8a3d01bd997a639cc6c5eaa51.